app/middleware/error_handler.js

```diff
@@ -1,21 +1,37 @@
 const httpErrors = require('http-errors');
 const httpStatus = require('http-status');
-const { pick } = require('lodash');
+const { ValidationError } = require('express-validation');
+const { pick, get } = require('lodash');
 
-const getErrorStatusCode = err =>
-  err.status ||
-  err.statusCode ||
-  httpStatus.INTERNAL_SERVER_ERROR;
+const getErrorStatusCode = (err) => {
+  if (err instanceof ValidationError) {
+    return 400;
+  }
+
+  return err.status ||
+    err.statusCode ||
+    httpStatus.INTERNAL_SERVER_ERROR;
+};
 
 const notFound = () => {
   throw new httpErrors.NotFound('Resource not found');
 };
 
+const prepareValidationError = err => ({
+  message: get(err, 'statusText', 'Expect valid parameters'),
+  errors: get(err, 'errors', []).map(error => ({
+    location: get(error, 'location'),
+    field: get(error, 'field[0]'),
```

Comment:
why field[0]? I think here you should use `get(error, 'field')`

Reply:
If I pick `field` value response will be as on picture(`field` will be an array)
![image](https://user-images.githubusercontent.com/9417851/39618530-d64b94f6-4f8c-11e8-864d-b9f722fef8f9.png)
That's why I pick only first item.
It was changed in [this](https://github.com/hapijs/joi/commit/8565afc5816ace1071dd5b1fa5a72355c6ec7df2#diff-f2d4d889f420f9ef17c4501ffd84e17f) commit in Joi library

Reply:
gotcha